### ISSUE TITLE
Who has access to the rbtree gem on RubyGems?

### ISSUE THREAD

**Issue by kyrylo** [zzak/rbtree#2]
Hi there @zzak!

I saw that someone pushed a new version of rbtree ([v0.4.4](https://rubygems.org/gems/rbtree/versions/0.4.4)) on January 1, 2021.
Was it you? If so, how did you get access/rights to push the gem under the name `rbtree`?

The owner of the gem is this profile: https://rubygems.org/profiles/lvfl98xod5hkd1mczt9yiuyoulblcy5z
Which, I believe, belongs to Ozawa Takuma.

Three years ago I made a [fork](https://github.com/kyrylo/rbtree3) of rbtree and released it as rbtree3 because I couldn't just push to `rbtree`.

I still maintain it but I am not sure who uses that besides me. It also includes some of the fixes you did here. Not sure if you saw my fork or https://github.com/skade/rbtree, where I was able to obtain the source code. @skade [let me know](https://github.com/skade/rbtree/pull/2#issuecomment-462269987) that he also doesn't have access to the `rbtree` gem on RubyGems.

**Comment by dentarg** [comment]
Looks like @mame is the one pushing `rbtree` to RubyGems.org? https://github.com/mame/rbtree

**Comment by kyrylo** [comment]
Oh wow, so now we have one more rbtree fork, which leads to more confusion.

Great find, @dentarg 😁🤝

**Comment by mame** [comment]
Yes, I pushed rbtree 0.4.5 and 0.4.6. Ozawa Takuma gave me a push privilege when I sent him a couple of patches to fix some build errors.

Note that the owner of rbtree gem is still Ozawa-san. When I need to make changes to rbtree, I email the patch to him for review before committing.
